Algorithm/QCAlgorithm.Python.cs

```diff
@@ -44,6 +44,8 @@ public partial class QCAlgorithm
         /// <summary>
         /// Sets pandas converter
         /// </summary>
+        [DocumentationAttribute(ConsolidatingData)]
+        [DocumentationAttribute(HandlingData)]
         public void SetPandasConverter()
```

Comment:
I don't think the users should need to be calling this method, so I think we might not document it really?